perfmetrics/scripts/run_e2e_tests.sh

```diff
@@ -25,6 +25,17 @@ readonly BUCKET_LOCATION="us-west1"
 # true or false to run e2e tests on installedPackage
 run_e2e_tests_on_package=$1
 
+# Upgrade gcloud version.
+# Kokoro machine's outdated gcloud version prevents the use of the "managed-folders" feature.
+gcloud version
+wget -O gcloud.tar.gz https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz -q
+sudo rm -rf $(which gcloud) && sudo tar xzf gcloud.tar.gz && sudo mv google-cloud-sdk /usr/local
+sudo /usr/local/google-cloud-sdk/install.sh
+export PATH=$PATH:/usr/local/google-cloud-sdk/bin
+echo 'export PATH=$PATH:/usr/local/google-cloud-sdk/bin' >> ~/.bashrc
+gcloud version && rm gcloud.tar.gz && gcloud components update
+sudo /usr/local/google-cloud-sdk/bin/gcloud components install alpha
```

Comment:
`sudo gcloud components install alpha` should also work

Reply:
`sudo gcloud ` will return this error`sudo: gcloud: command not found` .
Without sudo it is referring to use full path

```
ERROR: (gcloud.components.install) You cannot perform this action because you do not have permission to modify the Google Cloud SDK installation directory [/home/tulsishah_google_com/google-cloud-sdk].

Re-run the command with sudo: sudo /home/tulsishah_google_com/google-cloud-sdk/bin/gcloud ...
```

Reply:
`Also can we move this to after setting the architecture (line 40)?` May I know the reason behind this? as far as I know `architecture` was used for downloading go pkg.